# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/performance-timeline/supportedEntryTypes.any.js.ini
# Commit: e04ef5171459
# Full Hash: e04ef517145991e2aee797e935f43b7be35ff7aa
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-09 09:54:40
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25426
# ==============================================================================

diff -r ffaff6fb1fe3 -r e04ef5171459 testing/web-platform/meta/performance-timeline/supportedEntryTypes.any.js.ini
--- a/testing/web-platform/meta/performance-timeline/supportedEntryTypes.any.js.ini	Mon Apr 08 19:00:31 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,9 +0,0 @@
-[supportedEntryTypes.any.html]
-  [supportedEntryTypes exists and returns entries in alphabetical order]
-    expected: FAIL
-
-
-[supportedEntryTypes.any.worker.html]
-  [supportedEntryTypes exists and returns entries in alphabetical order]
-    expected: FAIL
-